tmail-backend/rate-limiter/rate-limiter-api/src/main/scala/com/linagora/tmail/rate/limiter/api/CacheRateLimitingPlan.scala

```diff
@@ -0,0 +1,52 @@
+package com.linagora.tmail.rate.limiter.api
+
+import com.github.benmanes.caffeine.cache.{Cache, Caffeine}
+import com.linagora.tmail.rate.limiter.api.CacheRateLimitingPlan.{CACHE_EXPIRE_DURATION, LIST_KEY}
+import org.reactivestreams.Publisher
+import reactor.core.scala.publisher.{SFlux, SMono}
+
+import java.lang
+import java.time.Duration
+
+object CacheRateLimitingPlan {
+  val CACHE_EXPIRE_DURATION: Duration = Duration.ofHours(2)
```

Comment:
The problem of a cache is cache invalidation. If the value is updated the cache will continue serving the outdated value.

In our case, this is acceptable, but only for small amount of time (IMO up to 2 minutes).

 -> We should make this configurable (from the mail configuration?)
 -> We likely should choose a way lower default (2 mins)